pkg/controller/controller.go

```diff
@@ -1288,8 +1290,11 @@ func (c *FrameworkController) deleteFramework(
 		"[%v]: Failed to delete Framework %v: confirm: %v: ",
 		f.Key(), f.UID, confirm)
 
+	ctx := context.TODO()
 	deleteErr := c.fClient.FrameworkcontrollerV1().Frameworks(f.Namespace).Delete(
-		f.Name, &meta.DeleteOptions{
+		ctx,
```

Comment:
>ctx [](http://example.com/codeflow?start=2&length=3)

I checked K8s offical controllers, seems ctx no need to reuse, so here just "context.TODO"?

Reply:
The follow up change may either pass in a ctx or create one as above lines. Both of them will keep ctx in following usage so I will leave as current version.